src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,24 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MainApp {
+    public static void main(String[] args) {
+
+        Machine truck = new Truck("Mikki");
+        Machine bulldozer = new Bulldozer("Eduardo");
+        Machine excavator = new Excavator("Maks");
+
+        Machine[] machines = {truck, bulldozer, excavator};
+        printTotal(machines);
+        System.out.println();
```

Comment:
There's a redundant empty line after the call to `printTotal(machines)` method. According to the checklist, we should avoid adding unnecessary empty lines after method calls.